tests/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ if(GMOCK_FOUND)
     cuke_add_test(integration/HookRegistrationTest)
     cuke_add_test(integration/StepRegistrationTest)
     cuke_add_test(integration/TaggedHookRegistrationTest)
-    if (NOT MSVC)
+    if(NOT WIN32)
```

Comment:
Yes I tried to run it and it failed. I suppose for the same reason as when built with MSVC, but I didn't investigate further. Maybe @paoloambrosio knows why this test fails (under MSVC)